src/functionalTest/java/uk/gov/hmcts/reform/payments/client/functional/CreatePaymentTest.java

```diff
@@ -46,9 +46,20 @@
     private static final String SITE_ID = "AA00";
 
     @Test
-    void canCreateAndRetrievePayments() {
+    void canCreateRetrieveAndCancelPayments() {
         User citizen = createCitizen();
-        PaymentDto createdPayment = paymentsClient.createPayment(
+        PaymentDto createdPayment = createPayment(citizen);
+
+        final String paymentGroupReference = createdPayment.getPaymentGroupReference();
+        final String reference = createdPayment.getReference();
+
+        retrievePayment(citizen, createdPayment, paymentGroupReference, reference);
+
+        paymentsClient.cancelPayment(citizen.getAuthToken(), reference);
```

Comment:
Do we need any assertions to prove that this has called the correct API?

Reply:
Will retrieve again and verify content post-cancellation